dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -792,11 +792,7 @@ public String getLocaleFacetLabelName(String facetLabelName, String facetCategor
     }
 
     private String getDatasetFieldFacetCategoryName(DatasetFieldType matchedDatasetField) {
-        if (matchedDatasetField.isFacetable()) {
-            if (matchedDatasetField.getParentDatasetFieldType() != null) {
-                return matchedDatasetField.getLocaleTitleWithParent();
-            }
-
+        if (matchedDatasetField.isFacetable() && !matchedDatasetField.hasParent()) {
             return getStringFromBundle(format(FACETBUNDLE_MASK_VALUE, matchedDatasetField.getName()));
```

Comment:
We can't be sure that extra label `facets.search.fieldtype.%s.label` will exists. Consider for example situations with custom metadata blocks